hack/verify-depstat.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+kube::golang::setup_env
+
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
+
+echo 'installing depstat'
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
+  GO111MODULE=on go install github.com/RinkiyaKeDad/depstat
+popd >/dev/null
+
+cd "${KUBE_ROOT}"
+
+# <(...) is process substitution which converts output of a 
+# command into a file like object which is what diff expects
+# -s reports if the files are same
+diff -s ${KUBE_ROOT}/dependency-stats.json <(depstat stats --json)
```

Comment:
Can you also add a `echo 'Verifying depstat'` above this line?

When I ran the script, I got a little confused for a second that script got stuck after fetching go modules (because depstat takes a while to run). This is probably because of me missing coffee :laughing: but documenting more steps won't hurt :)